diesel/Cargo.toml

```
@@ -35,7 +35,7 @@ r2d2 = { version = ">= 0.8, < 0.9", optional = true }
 
 [dev-dependencies]
 cfg-if = "0.1.0"
-dotenv = ">=0.8, <0.11"
+dotenv = "=0.15"
```

Is there any reason to use such a strict version range here? 
```suggestion
dotenv = "0.15"
```
should be fine, right?